cscs-checks/microbenchmarks/mpi/osu/osu_tests.py

```diff
@@ -194,36 +235,37 @@ def __init__(self):
 
 @rfm.simple_test
 class P2PCPULatencyTest(P2PBaseTest):
-    def __init__(self):
-        super().__init__()
-        self.valid_systems = ['daint:gpu', 'daint:mc', 'dom:gpu', 'dom:mc',
-                              'arolla:cn', 'tsa:cn', 'eiger:mc', 'pilatus:mc']
-        self.executable_opts = ['-x', '100', '-i', '1000']
-
-        self.executable = './p2p_osu_latency'
-        self.reference = {
-            'daint:gpu': {
-                'latency': (1.30, None, 0.70, 'us')
-            },
-            'daint:mc': {
-                'latency': (1.61, None, 0.85, 'us')
-            },
-            'dom:gpu': {
-                'latency': (1.138, None, 0.10, 'us')
-            },
-            'dom:mc': {
-                'latency': (1.24, None, 0.15, 'us')
-            },
-            'eiger:mc': {
-                'latency': (2.33, None, 0.15, 'us')
-            },
-            'pilatus:mc': {
-                'latency': (2.33, None, 0.15, 'us')
-            },
-            # keeping as reference:
-            # 'monch:compute': {
-            #     'latency': (1.27, None, 0.1, 'us')
-            # },
+    executable = './p2p_osu_latency'
```

Comment:
valid_systems missing ?

Reply:
you are right, I deleted by mistake, thanks!